src/OrchardCore/OrchardCore.DisplayManagement.Liquid/TemplateOptionsConfigurations.cs

```diff
@@ -55,13 +60,13 @@ public void Configure(TemplateOptions options)
                 return new ArrayValue(forms.Keys.Select(x => new StringValue(x)).ToArray());
             }
 
-            return new ArrayValue(forms[name].Select(x => new StringValue(x)).ToArray());
+            return new StringValuesValue(forms[name]);
         });
 
         options.MemberAccessStrategy.Register<HttpContextItemsWrapper, object>((httpContext, name) => httpContext.Items[name]);
-        options.MemberAccessStrategy.Register<QueryCollection, string[]>((queries, name) => queries[name].ToArray());
+        options.MemberAccessStrategy.Register<QueryCollection, StringValues>((queries, name) => queries[name]);
```

Comment:
Does this change the behavior mentioned here? https://github.com/OrchardCMS/OrchardCore/issues/1924#issuecomment-2588910251

Reply:
Yes, `StringValuesValue` allows treating multiple query parameters, form values, or headers as a single string in Liquid templates. For example:

```liquid
{% if Request.Query.Greeting == "helloworld" %}
Hello, world!
{% endif %}
```

This matches both `?Greeting=helloworld` and `?Greeting=hello&Greeting=world`.